clients/python/src/objectstore_client/auth.py

```diff
@@ -0,0 +1,56 @@
+from datetime import UTC, datetime, timedelta
+from enum import StrEnum
+from typing import Self
+
+import jwt
+
+from objectstore_client.scope import Scope
+
+
+class Permission(StrEnum):
+    """
+    Enum listing permissions that Objectstore tokens may be granted.
+    """
+
+    ObjectRead = "object.read"
+    ObjectWrite = "object.write"
+    ObjectDelete = "object.delete"
```

Comment:
Conventions suggest these constants as uppercase


```suggestion
    OBJECT_READ = "object.read"
    OBJECT_WRITE = "object.write"
    OBJECT_DELETE = "object.delete"
```